docs-source/spring/content/development/project.md

```diff
@@ -0,0 +1,205 @@
+            <scope>import</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-dependencies</artifactId>
+            <version>${spring-cloud.version}</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+    </dependencies>
+</dependencyManagement>    
+```
+
+### Spring Application Configuration
+
+Oracle recommends the following configuration to configure the application access to
+use the built-in services, including the Spring Eureka Service Registry and the
+observability tools.
+
+```yaml
+spring:
+  application:
+    name: account
+  zipkin:
+    base-url: ${zipkin.base-url}
+
+eureka:
+  instance:
+    hostname: ${spring.application.name}
+    preferIpAddress: true
+  client:
+    service-url:
+      defaultZone: ${eureka.service-url}
+    fetch-registry: true
+    register-with-eureka: true
+    enabled: true
+    
+management:
+  endpoint:
+    health:
+      show-details: always
+  endpoints:
+    web:
+      exposure:
+        include: "*"
+  metrics:
+    tags:
+      application: ${spring.application.name}
+```
+
+The variables in this configuration will be automatically injected to your deployment
+and pods when you use the Oracle Backend for Spring Boot and Microservices CLI to deploy your application. 
+
+#### Data Sources
+
+If your application uses a data source, you should add the following configuration.  Note that this
+example shows JPA, if you are using JDBC you should use the appropriate configuration.
+
+```yaml
+spring:
+  jpa:
+    hibernate:
+      ddl-auto: validate
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.OracleDialect
+        format_sql: false
+    show-sql: false
+
+  datasource:
+    url: ${spring.datasource.url}
+    username: ${spring.datasource.username}
+    password: ${spring.datasource.password}
+    driver-class-name: oracle.jdbc.OracleDriver
+    type: oracle.ucp.jdbc.PoolDataSource
+    oracleucp:
+      connection-factory-class-name: oracle.jdbc.pool.OracleDataSource
+      connection-pool-name: AccountConnectionPool
+      initial-pool-size: 15
+      min-pool-size: 10
+      max-pool-size: 30
+```
+
+The variables in this configuration will be automatically injected to your deployment
+and pods when you use the Oracle Backend for Spring Boot and Microservices CLI to deploy your application. 
+
+#### Liquibase
+
+If you are using Liquibase to manage your database schema and data, you should
+add the following dependency:
+
+```xml
+<properties>
+    <liquibase.version>4.24.0</liquibase.version>
+</properties>
+
+<dependencies>
+    <dependency>
+        <groupId>org.liquibase</groupId>
+        <artifactId>liquibase-core</artifactId>
+        <version>${liquibase.version}</version>
+    </dependency>
+</dependencies>
+```
+
+The following configuration should be added to your Spring application configuration:
```

Comment:
Change to "Add the following configuration to your Spring application configuration:"